spmd/tensor/ops/pointwise_ops.py

```diff
@@ -20,6 +20,7 @@
 
 
 linear_pointwise_ops = [
+    "aten.contiguous.default",
```

Comment:
Could we make this a tensor_ops instead? I feel we don't need to do resharding for contiguous(), it should just preserve the existing sharding?

Reply:
oh of course, here, right? https://github.com/pytorch/tau/blob/main/spmd/tensor/ops/tensor_ops.py#L80

Reply:
Yeah that should be the right place